dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/plugin/PrioritySPI.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.spi.plugin;
+
+public interface PrioritySPI extends Comparable<Integer> {
+
+    SPIIdentify getIdentify();
+
+    /**
+     * Get the conflict strategy of the spi plugin, this will be used when there are existed two plugin has the same identify name.
+     *
+     * <p>
+     * The default strategy will use the high priority plugin.
+     */
+    default SPIConflictStrategy getConflictStrategy() {
+        return SPIConflictStrategy.LOAD_HIGH_PRIORITY;
+    }
+
+    @Override
+    default int compareTo(Integer o) {
+        return Integer.compare(getIdentify().getPriority(), o);
+    }
+
+    enum SPIConflictStrategy {
```

Comment:
I think we just support the `LOAD_HIGH_PRIORITY` strategy is enough, if two plugin has same priority, throw exception. Which can make a more clear load rule and make users understand earsier.

Reply:
Need to add doc about the SPI load rule.

Reply:
Yes, agree with you

Reply:
I find we have missing the doc about SPI, I just add doc in the code, we need to add the introduction about our SPI, this should be another PR.

Reply:
The SPI doc had exist,see https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/contribute/backend/spi/task.html

Reply:
Thanks, I add the doc about plugin priority.